Backend/dev/migrations/dynamic-offer-driver-app/0148-add-ride-location-table-and-columns-to-ride-table.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE atlas_driver_offer_bpp.trip_location as SELECT * FROM atlas_driver_offer_bpp.booking_location;
```

Comment:
Don't do this refactoring, why are using changing location table name? This is not backward compatible.

Reply:
cc : @NikithShetty @sidsethujuspay